conf/messages

```diff
@@ -3039,15 +3085,19 @@ annualExemptAmount.agent.title=How much of your client’s Capital Gains Tax Ann
 annualExemptAmount.trust.title=How much of the trust’s Capital Gains Tax Annual Exempt Amount does it want to use?
 annualExemptAmount.capacitor.title=How much of the person’s Capital Gains Tax Annual Exempt Amount do they want to use?
 annualExemptAmount.personalRep.title=How much of the person’s Capital Gains Tax Annual Exempt Amount do they want to use?
+annualExemptAmount.personalRepInPeriodOfAdmin.title=How much of the Annual Exempt Amount are you including?
+annualExemptAmount.personalRepInPeriodOfAdmin.agent.title=How much of the Annual Exempt Amount are you including?
 
 annualExemptAmount.helpText=In the tax year {0} to {1}, you only have to pay Capital Gains Tax on gains above {3}.
 annualExemptAmount.agent.helpText=In the tax year {0} to {1}, your client only has to pay Capital Gains Tax on gains above {3}.
 annualExemptAmount.trust.helpText=In the tax year {0} to {1}, most trustees only have to pay Capital Gains Tax on gains above {2}. This is called the Annual Exempt Amount. <br/> <br/>The Annual Exempt Amount for trusts for vulnerable beneficiaries is {3}.
 annualExemptAmount.capacitor.helpText=In the tax year {0} to {1}, the person only has to pay Capital Gains Tax on gains above {3}.
 annualExemptAmount.personalRep.helpText=In the tax year {0} to {1}, the person only has to pay Capital Gains Tax on gains above {3}.
+annualExemptAmount.personalRepInPeriodOfAdmin.helpText=During the period of administration, all of the Annual Exempt Amount can be included in the estate's return. This is the case even if the time between 6 April and the day the person died was very short.
+annualExemptAmount.personalRepInPeriodOfAdmin.agent.helpText=During the period of administration, all of the Annual Exempt Amount can be included in the estate's return. This is the case even if the time between 6 April and the day the person died was very short.
 
 annualExemptAmount.details.1.header=What is the Annual Exempt Amount?
-annualExemptAmount.details.1.body=Each tax year a trust gets an annual tax-free allowance. This is known as the <a href="{0}" target="_blank"> Annual Exempt Amount (opens in a new window)</a>. Trustees only pay Capital Gains Tax if the trust’s taxable gains for the tax year are above the Annual Exempt Amount.
+annualExemptAmount.details.1.body=The <a href="{0}" target="_blank"> Annual Exempt Amount (opens in a new window)</a> is an annual tax-free allowance. It is available to certain people who need to pay Capital Gains Tax. Most people only have to pay Capital Gains Tax on their gains for the tax year above the Annual Exempt Amount. Any losses or reliefs must have been subtracted from the gain already.
```

Comment:
it looks like the old content for trusts should be there when the user is a trust - this new content is for period of admin only

Reply:
fixed - added another set of header and body and tagged it with `3` - not sure if that is the convention that you are using here